scalableminds/webknossos
# Renamed soma clicking

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open skeleton tracing and check if soma clicking was replaced with "Create single-node trees"

### Issues:
- fixes #3119

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
-  ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
-  ~~~[ ] Needs datastore update after deployment~~~
